code/style.css

```diff
@@ -1,7 +1,500 @@
+   position: absolute;
+   top: 0;
+   left: 0;
+   height: 16px;
+   width: 16px;
+   background-color: #fff;
+   border: 1px solid rgba(51, 51, 51, 0.575);
+ }
+
+ .checkbox:hover input~.checkmark {
+   background-color: #ccc;
+   border-color: teal;
+ }
+
+ /* glow on focus */
+
+ input:focus~.checkmark {
+   border-color: teal;
+   box-shadow: 0px 0px 4px teal;
+ }
+
+ /* creation of indicator hidde when not checked */
+
+ .checkmark:after {
+   content: "";
+   position: absolute;
+   display: none;
+ }
+
+ /* showing the checkmark when checked */
+
+ .checkbox input:checked~.checkmark:after {
+   display: block;
+ }
+
+ /* checkmark indicator */
+
+ .checkbox .checkmark:after {
+   left: 5px;
+   top: 1px;
+   width: 4px;
+   height: 8px;
+   border: 1px solid teal;
+   border-width: 0 3px 3px 0;
+   -webkit-transform: rotate(45deg);
+   -ms-transform: rotate(45deg);
+   transform: rotate(45deg);
+ }
+
+ /******* Form buttons *******/
+
+ .align-buttons {
+   display: flex;
+   flex-direction: row;
+ }
+
+ .form-button {
+   display: flex;
+   align-self: center;
+   color: rgba(51, 51, 51, 0.822);
+   max-width: 100px;
+   margin-top: 30px;
+   padding: 10px;
+   cursor: pointer;
+   border: 2px solid teal;
+   background-color: rgba(255, 255, 255, 0.555);
+ }
+
+ .form-button:hover {
+   transform: scale(1.2);
+   transition: 0.1s;
+ }
+
+ .submit-button {
+   margin-right: 20px;
+ }
+
+ .clear-button {
+   margin-right: 20px;
+ }
+
+ .clear-button:hover {
+   border: 2px solid goldenrod;
+ }
+
+ /****************** Accordion styles **************/
+
+ .accordion-inner-wrapper {
+   display: flex;
+   flex-direction: column;
+   justify-content: center;
+ }
+
+ .accordion-button-styling {
+   border: none;
+   background-color: rgba(255, 255, 255, 0.932);
+   cursor: pointer;
+   padding: 18px 10px;
+   text-align: left;
+   outline: none;
+ }
+
+ .accordion h1 {
+   display: flex;
+   align-self: center;
+   justify-content: center;
+   font-weight: 500;
+ }
+
+ .active {
+   background-color: teal !important;
+   color: #fff;
+ }
+
+ .question:hover {
+   background-color: teal;
+ }
+
+ .answer {
+   padding: 0 18px;
+   background-color: white;
+   display: none;
+   overflow: hidden;
+   line-height: 28px;
+ }
+
+ .question:after {
+   content: '\02795';
+   font-size: 13px;
+   color: #777;
+   float: right;
+   margin-left: 5px;
+ }
+
+ div.section:nth-child(even) .question {
+   background-color: #dee2e6;
+ }
+
+ div.section:nth-child(even):hover .question {
+   background-color: teal;
+ }
+
+ .active:after {
+   content: '\2796';
+ }
+
+ .active+.answer {
+   display: inline-block;
+   animation: active 1s;
+ }
+
+ /* animation on slide */
+
+ @keyframes active {
+   0% {
+     display: none;
+     transform: translate3d(0, -1rem, 0);
+     opacity: 0;
+   }
+   1% {
+     display: block;
+     transform: translate3d(0, -1rem, 0);
+     opacity: 0;
+   }
+   100% {
+     transform: translate3d(0, 0, 0);
+     opacity: 1;
+   }
```

Comment:
I have never used keyframes but Im guessing this is to make the slide in the accordion. 
Could this have been done without keyframes?

Reply:
exactly ) 
Honestly no idea , I think it can also be done with use of canvas html tag through JS , but haven't tried it yet/ read about .